stake-pool/program/src/instruction.rs

```diff
@@ -31,12 +31,13 @@ pub enum StakePoolInstruction {
     ///
     ///   0. `[w]` New StakePool to create.
     ///   1. `[s]` Owner
-    ///   2. `[w]` Uninitialized validator stake list storage account
-    ///   3. `[]` pool token Mint. Must be non zero, owned by withdraw authority.
-    ///   4. `[]` Pool Account to deposit the generated fee for owner.
-    ///   5. `[]` Clock sysvar
-    ///   6. `[]` Rent sysvar
-    ///   7. `[]` Token program id
+    ///   2. `[]` Manager
+    ///   3. `[w]` Uninitialized validator stake list storage account
+    ///   4. `[]` Pool token mint. Must be non zero, owned by withdraw authority.
```

Comment:
who is `withdraw authority` here?

Reply:
That's the stake pool program derived address, the actual withdraw authority on all stake accounts in the pool.